pkgs/development/python-modules/dash/default.nix

```diff
@@ -107,20 +109,32 @@ buildPythonPackage rec {
   };
 
   nativeCheckInputs = [
+    flaky
     pytestCheckHook
     pytest-mock
     mock
+    numpy
     pyyaml
-  ];
+  ] ++ lib.flatten (builtins.attrValues optional-dependencies);
 
   disabledTestPaths = [
     "tests/unit/test_browser.py"
     "tests/unit/test_app_runners.py" # Uses selenium
     "tests/integration"
+    # dash_test_components is not available
```

Comment:
How hard is it to package those from https://github.com/plotly/dash/tree/v3.2.0/%40plotly/dash-test-components?